python/sparktk/models/clustering/gmm.py

```diff
@@ -152,10 +152,10 @@ class GaussianMixtureModel(PropertiesObject):
         >>> assert(np.allclose(expected_sigma, actual_sigma, atol=1e+01))
         </hide>
 
-        >>> model.predict(frame)
+        >>> predicted_frame =  model.predict(frame)
 
         <skip>
-        >>> x = frame.take(9)
+        >>> x = predicted_frame.take(9)
```

Comment:
I'm thinking this is supposed to be predicted_frame.inspect(), not take, because the take() with assignment wouldn't print out anything.  It's in the skip block, so it's not failing, even though it's wrong.